### PR TITLE
Add TMC StealthChop toggle for DWIN display

### PR DESCRIPTION
**Pull request by wlkmanist**
### Description

Adds StealthChop controls to TMC menu

### Requirements

DWIN display, TMC drivers with UART

### Benefits

More settings

### CODE REVIEW

**Review by classicrocker883 — APPROVED** [review]

**Comment by classicrocker883** [comment]
hi @wlkmanist, could you have a look at this **PR**?
https://github.com/mriscoc/Ender3V2S1/pull/1276

it is UART related, I am wondering if I should incorporate this as a fix here. if so, would you mind writing up the pull request and perhaps let me know if this is a good fix as well?

**Comment by wlkmanist** [comment]
> hi @wlkmanist, could you have a look at this **PR**? [mriscoc#1276](https://github.com/mriscoc/Ender3V2S1/pull/1276)
> 
> it is UART related, I am wondering if I should incorporate this as a fix here. if so, would you mind writing up the pull request and perhaps let me know if this is a good fix as well?

Hi. I think I can't properly understand your question because of my bad english.
I looked at this PR and I can't test it on my setup, at least for now. I will solder my board later and will try this then.

**Comment by classicrocker883** [comment]
oh no, you don't have to do that if you do not want to.
I was asking since you use TMC drivers with UART, yes?
I thought that PR was related and could be used in this repository.

if you must solder I ask you do not worry about it. I was asking you could look at the code changes and make another pull request. or let me know if I should copy that code. Will it make tmc uart better?

**Comment by wlkmanist** [comment]
> oh no, you don't have to do that if you do not want to. I was asking since you use TMC drivers with UART, yes? I thought that PR was related and could be used in this repository.
> 
> if you must solder I ask you do not worry about it. I was asking you could look at the code changes and make another pull request. or let me know if I should copy that code. Will it make tmc uart better?

I am using UART on extruder driver with TMC2208. And I am planning to make uart on Z and maybe on other axis. So I can test it, but not now.

File: Marlin/src/lcd/language/language_en.h [hunk 1]
```diff
@@ -1042,6 +1043,7 @@ namespace LanguageWide_en {
     LSTR MSG_TMC_BSTEALTH                   = _UxGT(STR_B " " STR_TMC_STEALTH);
     LSTR MSG_TMC_CSTEALTH                   = _UxGT(STR_C " " STR_TMC_STEALTH);
     LSTR MSG_TMC_ESTEALTH                   = _UxGT(STR_E " " STR_TMC_STEALTH);
+    LSTR MSG_TMC_ENSTEALTH                  = _UxGT(* " " STR_TMC_STEALTH);
```
Comment (wlkmanist) [inline Marlin/src/lcd/language/language_en.h:1046]: Compile error at this line

Comment (wlkmanist) [reply Marlin/src/lcd/language/language_en.h:1046]:
```suggestion
    LSTR MSG_TMC_ENSTEALTH                  = _UxGT("* " STR_TMC_STEALTH);
```

**Comment by wlkmanist** [comment]
> hi @wlkmanist, could you have a look at this **PR**? [mriscoc#1276](https://github.com/mriscoc/Ender3V2S1/pull/1276)

I soldered my board, now all drivers in UART mode. Have not experienced any issues with UART and compilation. So I can't say if this PR useful or not. Works fine even without it.

**Pull request merged**